collect_app/src/main/java/org/odk/collect/android/tasks/sms/models/SubmitFormModel.java

```diff
@@ -0,0 +1,50 @@
+package org.odk.collect.android.tasks.sms.models;
+
+public class SubmitFormModel {
```

Comment:
Is this abstraction all that helpful? I would tend to remove it. It's not SMS-specific and has a lot of overlap with `SmsSubmission`. All the other info can be pulled from the DB when needed. `submitForm` can take in `formId` and `formVersion` so that it can still be tested.

Reply:
Consider also looking at the existing `FormInfo` class.

Reply:
`FormInfo` is very useful. I hadn't seen it that's why I created this model. I was trying to pass in all the data required in `submitForm` to limit the number of database calls necessary since the `InstancesDao' was returning all the information needed. Using this model made it a bit easier to supply test-related properties instead of having to mock more calls to the `InstancesDao`.